test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -198,7 +197,10 @@ Describe "Verify approved aliases list" -Tags "CI" {
         # We control only default engine aliases (Source -eq "") and aliases from following default loaded modules
         $moduleList = @("Microsoft.PowerShell.Utility", "Microsoft.PowerShell.Management")
         $currentAliasList = Get-Alias | Where-Object { $_.Source -eq "" -or $moduleList -contains $_.Source } | Select-Object -ExpandProperty DisplayName
-
-        Compare-Object -ReferenceObject $currentAliasList -DifferenceObject $aliaslist | Should Be $null
+        $result = Compare-Object -ReferenceObject $currentAliasList -DifferenceObject $aliaslist
+        # Below 'Should Be' don't show full list wrong aliases so we output them explicitly
```

Comment:
We should remove this from `Default-Aliases.Tests.ps1` as the last step in the PR.